packages/js/components/src/plugins/index.js

```diff
@@ -18,7 +18,6 @@ export class Plugins extends Component {
 		};
 
 		this.installAndActivate = this.installAndActivate.bind( this );
-		this.skipInstaller = this.skipInstaller.bind( this );
```

Comment:
Can we still get the correct `this.` in `skipInstaller`?

Reply:
Whoops, this was accidental. Thanks for spotting!